### ISSUE TITLE
Stats bar is shrinked in NTP in an upgraded profile

### ISSUE THREAD

**Issue by GeetaSarvadnya** [brave/brave-browser#24409]
## Description 

Stats bar is shrinked in NTP in an upgraded profile

## Steps to reproduce 

  No definite steps to reproduce the issue as it reproduced only once. 

## Actual result 
![image](https://user-images.githubusercontent.com/38657976/182570092-d60de4ba-7fa2-4f5d-894a-a8123f7cb0a4.png)

## Expected result

Stats bar shoud not shrinked in NTP 
## Issue reproduces how often 
Reproduce only once

## Version/Channel Information:

- Can you reproduce this issue with the current Play Store version? NA
- Can you reproduce this issue with the current Play Store Beta version? NA
- Can you reproduce this issue with the current Play Store Nightly version?NA

## Device details
   - Install type (ARM, x86): ARM
   - Device type (Phone, Tablet, Phablet): Samsung Galaxy Tab A7
   - Android version: 1.42.86
   
## Brave version
1.42.86

### Website problems only
- Does the issue resolve itself when disabling Brave Shields? NA
- Does the issue resolve itself when disabling Brave Rewards? NA
- Is the issue reproducible on the latest version of Chrome? NA

### Additional information

cc: @brave/qa-team @SergeyZhukovsky @samartnik @anthonypkeane

**Comment by bsclifton** [comment]
Closing as stale - let's create a new issue if this is still a problem
